backend/ctrl.js

```diff
@@ -75,8 +75,53 @@ const getContacts = async (req, res) => {
   }
 };
 
+const newContact = async (req, res) => {
+  try {
+    const db = await req.app.get('db');
+    const { firstName, lastName, phone, email, address1, address2, city, state, zip } = req.body;
+
+    await db.contacts.insert({
+      first_name: firstName,
+      last_name: lastName,
+      phone: phone,
+      email: email,
+      address1: address1,
+      address2: address2,
+      city: city,
+      state: state,
+      zip: zip,
+      created_at: new Date(),
+      updated_at: new Date()
+    })
+
+    res.status(200).send('Successfully saved new contact!')
```

Comment:
You should also send back the object formatted in the way the front end uses it so that you can do an optimistic update and add the response into the list in redux. That way you don't need to make a `GET` afterwards to get the new value

Reply:
I am now sending back the saved contact, and am attempting to set it to `newContact` in the fulfilled step, but I might be missing something. Saving new contacts to the db still works though.